packages/ontario-design-system-global-styles/src/styles/scss/theme.scss

```
@@ -47,13 +47,8 @@
 
 /*** 6 - Components ***/
 @forward './6-components/buttons.component';
-@forward './6-components/form.component';
-@forward './6-components/labels.component';
-@forward './6-components/text-inputs.component';
-@forward './6-components/callouts-asides';
-@forward './6-components/page-alerts.component';
 @forward './6-components/all.component';
 
-/*** 7 - Overrides ***/
+/*** 7 - Overrides *å**/
```

A special character seems to have been introduced here.

```suggestion
/*** 7 - Overrides ***/
```